src/server/supplements/inspectorController.ts

```diff
@@ -45,15 +45,15 @@ export class InspectorController implements InstrumentationListener {
         case 'log':
           const originalMetadata = this._waitOperations.get(info.waitId)!;
           originalMetadata.log.push(info.message);
-          this.onCallLog('api', info.message, sdkObject, originalMetadata);
+          await this.onCallLog('api', info.message, sdkObject, originalMetadata);
```

Comment:
I think these were without await on purpose.

Reply:
should be .catch then?